src/app/client/src/app/modules/shared/components/collection-tree/collection-tree.component.ts

```diff
@@ -91,6 +90,15 @@ export class CollectionTreeComponent implements OnInit, OnChanges {
         node.icon = this.options.customFileIcon[node.fileType] || this.options.fileIcon;
         node.icon = `${node.icon} ${node.iconColor}`;
       }
+      if (node.folder && !(node.children.length)) {
+        node.title = node.model.name + '<strong> (Coming Soon!)</strong>';
```

Comment:
Coming Soon! should come from resource bundles

Reply:
Yeah, Used Resource service here.